runner/scheduler.py

```diff
@@ -263,7 +264,6 @@ def fast_forward_tests(self, tests: dict[str, TestExample]):
             match evt.type:
                 case EventType.BEGIN:
                     result, skip = self.initialise_result(test)
-                    assert not skip
```

Comment:
I added this "fix". It did fix resuming on the run with Claude, but I'm not 100% sure about it.

Reply:
Has it ever failed "legitimately"?

Reply:
Maybe? I don't know. Early fails were due to system failures, so many things were wrong. Now the system is more stable and it hasn't even failed so often during resuming.